Ghidra/Processors/upd78/LICENSE.txt

```diff
@@ -0,0 +1,8 @@
+Ghidra software is released under the Apache 2.0 license. In addition,
```

Comment:
We don't currently support third-party license files in the code repository. The license files will be automatically discarded during the build process. See https://github.com/NationalSecurityAgency/ghidra/pull/715 as an example.